### ISSUE TITLE
Simplify markup house style page

### ISSUE THREAD

**Issue by josebolos** [springernature/frontend-playbook#219]
In the [markup house style](markup/house-style.md) page there's a very long section explaining why we write HTML 5 (instead of XHTML). There are also several mentions of older versions of (X)HTML along the document, and how those old standards influence what we write today.

Although that whole page is beautifully written, it's 2018 and HTML 5 has been the standard for long enough that I'm concerned that many readers may not be familiar with XHTML or HTML 4 (or older!).

Is it necessary to explain why we use HTML 5? Could this page be improved by having a much much shorter and simpler explanation? I feel like it may feel less daunting to people getting started.

There's also a nicely written section about the HTML outline model. Considering this is directly related to accessibility and assistive technologies, should this section be included with the rest of the accessibility stuff (and adding links where necessary), instead of here?

**Comment by Alicole** [comment]
I would have thought we have now standardised on HTML 5 across various sites and products.

XHTML may still be used on some legacy pages, but HTML 5 has been the defacto standard for web development for a while now. So yes as you say it just adds to the confusion.

**Comment by jpw** [comment]
As HTML5 can still be written in a XML syntax & serialisation, and some people assume XML is desirable because it is more strict (not an unreasonable assumption), I'm keen to retain the reasoning to avoid anything XML being considered.

However I'd be happy to see any proposal reorganising or rewriting this doc, to make it less intimidating for those unprepared for the @jpw XML rant 😛 

> Considering this is directly related to accessibility and assistive technologies, should this section be included with the rest of the accessibility stuff (and adding links where necessary), instead of here?

Yes, but this is the markup house style page and I think should best live here? Maybe more cross-linking as you suggest?

**Comment by hollsk** [comment]
Oh, i forgot the outline section was on that page. That is interesting because i'm in the middle of the new a11y checklist and I was also thinking about how to include stuff about the outline algorithm there. This markup doc is super old, I think it might have come from a page in the old Nature wiki? It's about time for a refresh.

As i said, I wanted to put something about the outline in the refreshed a11y checklist. I don't want to have a huge amount of content on the checklist itself. I do want to link off to other bits of the playbook from the checklist. IMO we should keep the outline stuff here (as a deep dive), link to it from the a11y checklist, and make the whole page less scary 👻

**Comment by jpw** [comment]
![image](https://user-images.githubusercontent.com/683955/37470583-b9793170-285f-11e8-9a72-e4a6c5176179.png)

**Comment by jpw** [comment]
`interpret out instructions typo
